.github/workflows/dance.yml

```diff
@@ -57,6 +61,12 @@ jobs:
         with:
           cache-key: dance
 
+      # set up Python build environment
+      - name: Install Python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install pymongo
```

Comment:
I think `pip3 install pymongo` should work instead of two commands. Please check